### DISCUSSION TITLE
AnalyticalTable: How to set accessible name?

### DISCUSSION THREAD

**Discussion by stefanomantova** [UI5/webcomponents-react#4257] [Q&A] [answered]
When trying to set an accessible name on `AnalyticalTable` component, it is possible to see that this component currently miss both props `accessibleName` and `accessibleNameRef`. 

Is there any other way to enable screen readers (such as JAWS) to properly read the content and title of this component? These props are available for a significant quantity of components, including the regular `<Table/>`

**Comment by Lukas742** [comment, marked as answer]
Hi @stefanomantova 

since the `AnalyticalTable` is not a web-component, but a standard React component using web-components. It's no necessary to offer properties like `accessibleName` as you can just set the `aria-label` yourself.
